resources/public/api/conf/2.0/application.raml

```diff
@@ -616,4 +616,77 @@ traits:
                         description: The specified arrival does not exist, has been archived or is not available to the EORI number.
                         value:
                           code: NOT_FOUND
+        /{messageId}:
+          get:
+            displayName: Get a cached message related to a arrival and message ID
+            description: |
+              Retrieve all messages related to a specific arrival and message ID that have been received within the past 31 days.
+
+              For example, this could include ‘Declaration rejected’ or Movement Reference Number (MRN) allocated messages.
+
+              Movements older than 31 days and their related messages are archived and cannot be returned by the API.
 
+               The response can be returned in one of the following formats based on the provided `Accept` header:
+
+               * `application/vnd.hmrc.2.0+json-xml`: JSON-wrapped XML, where the message is provided as an XML string sent as part of JSON payload containing other metadata.
+               * `application/vnd.hmrc.2.0+json`: JSON, where the message is provided as a sub-document sent as part of a JSON payload containing other metadata.
```

Comment:
Remove this bit until we implement the json conversion for arrivals

Reply:
removed this